Typo in Javadoc for "io.pravega.client.stream.StreamConfiguration"

**Problem description**
./gradlew javadocs throws a warning 

/home/travis/build/pravega/pravega/client/src/main/java/io/pravega/client/stream/StreamConfiguration.java:45: warning - Tag @link: can't find getCurrentTimeWindow() in io.pravega.client.stream.EventStreamReader
(https://travis-ci.org/pravega/pravega/jobs/642682990?utm_medium=notification&utm_source=github_status)

**Problem location**
Javadoc for io.pravega.client.stream.StreamConfiguration -> timestampAggregationTimeout

**Suggestions for an improvement**
replace {@link EventStreamReader#getCurrentTimeWindow()} with {@link EventStreamReader#getCurrentTimeWindow(Stream)}

--- a/client/src/main/java/io/pravega/client/stream/StreamConfiguration.java
+++ b/client/src/main/java/io/pravega/client/stream/StreamConfiguration.java
@@ -39,7 +39,7 @@ public class StreamConfiguration implements Serializable {
      * The duration after the last call to {@link EventStreamWriter#noteTime(long)} which the
      * timestamp should be considered valid before it is forgotten. Meaning that after this long of
      * not calling {@link EventStreamWriter#noteTime(long)} the writer will be forgotten.
-     * If there are no known writers, readers that call {@link EventStreamReader#getCurrentTimeWindow()}
+     * If there are no known writers, readers that call {@link EventStreamReader#getCurrentTimeWindow(Stream)}
      * will receive a `null` when they are at the corresponding position in the stream.
      */
     private final long timestampAggregationTimeout;

